artifacts/mysql-statefulset.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Spec.Name }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Spec.Name }}
+  serviceName: {{ .Spec.Name }}
+  replicas: 1
```

Comment:
If we're introducing master/slave configuration, replica count should be configurable now.